CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - `realm-management {interfaces, triggers} sync`: do not show a wrong curl
   command when using `--to-curl`. Reference individual commands instead.
+- `appengine send-data`: fix the decoding of longinteger and longintegerarray
+  before sending to astarte. Those are no lomnger treated like interface arrays
```

Comment:
Typo (lomger --> longer), but I'd suggest to remove the "Those..." line, as it does not give much information to the user (it's more about the internals of astartectl)